core/trino-main/src/main/java/io/trino/operator/OutputSpoolingOperatorFactory.java

```diff
@@ -23,12 +23,12 @@
 import io.trino.server.protocol.OutputColumn;
 import io.trino.server.protocol.spooling.QueryDataEncoder;
 import io.trino.server.protocol.spooling.SpooledBlock;
-import io.trino.server.protocol.spooling.SpoolingManagerBridge;
 import io.trino.spi.Mergeable;
 import io.trino.spi.Page;
 import io.trino.spi.block.Block;
 import io.trino.spi.protocol.SpooledSegmentHandle;
 import io.trino.spi.protocol.SpoolingContext;
+import io.trino.spi.protocol.SpoolingManager;
```

Comment:
`Generate direct URIs to spooled segments` commit contains renames which are not important part of the change. Please extract those

Reply:
Actually whole this commit looks like refactors. What is important here if anything?

Reply:
Oh - ok. Impl of `directLocation()`